Sources/TuistLoader/Models+ManifestMappers/Headers+ManifestMapper.swift

```diff
@@ -32,17 +32,42 @@ extension TuistGraph.Headers {
                 return TuistGraph.Headers.extensions.contains(".\(fileExtension)") && !excluding.contains($0)
             }
         }
-        let `public`: [AbsolutePath] = try manifest.public?.globs.flatMap {
-            unfoldGlob(try generatorPaths.resolve(path: $0.glob), excluding: try resolveExcluding($0.excluding))
-        } ?? []
 
-        let `private`: [AbsolutePath] = try manifest.private?.globs.flatMap {
-            unfoldGlob(try generatorPaths.resolve(path: $0.glob), excluding: try resolveExcluding($0.excluding))
-        } ?? []
+        var autoExlcudedPaths = Set<AbsolutePath>()
+        let `public`: [AbsolutePath]
```

Comment:
I think rather than keep wrapping `public` and `private` into backticks, it might be better to opt-in for a more descriptive name such as `publicHeaders`? It does not really read well the way it is currently.